Fix roll in Merge PhysBone (#1291)
anatawa12/AvatarOptimizer: .docs/content/docs/reference/merge-physbone/index.md

For colliders, you can select `Merge` to merge colliders array from source physbones.

For Endpoint Position, you can select `Clear` to apply [Clear Endpoint Position](../clear-endpoint-position)

> For Limit Rotation, you can select `Fix` to fix different roll axis on the model.

Reviewer: 参考画像があると尚良いかも

Author: どういう画像を想定してますか？

Reviewer: 上手く統合できる対象の例です

Author: うーん手元に良い例がないのでむずい。あとどこを見せるかがわかってない
(インスペクター？)

Reviewer: Scene Viewですかね…分かりやすいユースケースはスカートとかだと思うので、こういうのっていう(それでもRollのみだと無理なケースはあるんでしょうけど)

Reviewer: PB の Angle Limitを付けると適用前後でGizmoで見える制限角度の見た目変わると思ったので…

Author: 適用はなにの適用前後？

Author: MergePBの適用前後という意味だと変わらないし、Fixにする前だとFixにしないとそもそもマージができない

Reviewer: > そもそもマージができない

Setで…?

Reviewer: まあ無くても良いかもです
伝わりそうな文言にはなったので

Author: SetはできるけどSetだと適切な価がないので実務上は無理

Author: なしにします

> If your model has different roll bones and fixes with Limit Rotation, you may use this option.

Reviewer: ```suggestion
If your model has different roll bones, you can limit their rotations together with this option.
```

Author: yosasou
